spec/spec_helper.rb

```diff
@@ -1 +1,9 @@
 require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet'
```

Comment:
Yes "bundle exec rake spec". Tested in my rvm environment with ruby 1.9.3 and 2.2.4